Topics/Tech_Stacks/swift.md

```diff
@@ -57,13 +58,71 @@ This information can be changed later, so for starters you can leave `Team` to b
 
 Make sure to use `SwiftUI` and `Swift` as your interface and language respectively, then click `Next` to choose where to store your project, and now you're ready to start.
 
+## Swift View
+A view is a part of a swift application’s user interface. Creating a new project automatically creates a view called content view.  
+
+<img width="283" alt="Screenshot 2023-11-23 at 12 32 33 AM" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/97854264/58e8197c-897b-4bff-be4d-77b6daa62ecd">
+
+Views in Swift are defined as structs and must conform to the View protocol. The content and behavior of the view are provided in the body of the view. To see how the view is transformed into the user’s interface and how users can interact with the view, we can either build and run our application on a simulated device or view the Xcode preview. Refer to the Testing Your App - Simulators: Background section to set up the simulator. An Xcode preview is created by the code defined in lines 22 - 24. It generates a view preview without having to build and run the app. When we open up our project, the preview will appear, as shown below, displaying the view.
```

Comment:
You could possibly add a link to the 'testing your app' portion of the file. It might make it easier for the reader to get there.

Reply:
Yes I was thinking about adding the link but I didn't yet. Thanks for reminding me!

Reply:
Would it be best if I merged my PR first so that you can add this link? I'll aim to add some images to my part later today and we could merge mine before yours?

Reply:
Yes I think that would be the best order to do it in!